HealthSDK/GiniHealthSDK/Documentation/source/Integration.md

```diff
@@ -110,3 +110,70 @@ config.loadingIndicatorColor = .black
 .
 healthSDK.setConfiguration(config)
 ```
+
+## Payment component view initialization
+We provide a custom payment component view meant to help user to pay the invoice/document.
+Payment component looks like this: 
+<br>
+<center><img src="img/Customization guide/PaymentComponentView.PNG" height="400"/></center>
+</br>
+
+##### Steps to integrate payment component view:
+
+##### 1. You need to instantiate a `PaymentComponentsController` class with a `GiniHealth` as parameter.
+```swift
+let paymentComponentsController = PaymentComponentsController(giniHealth: health)
+```
+
+##### 2. You need to pass this `paymentComponentsController` to the invoices/documents controller. 
+There you should have a list of invoices/documents that conform to this structure: `GiniDocument`. 
```

Comment:
[suggestion]: I would rewrite this like this:
There you should have a list of invoices/documents that conform to`GiniDocument` protocol.
How does this sound?